app/scripts/migrations/060.test.js

```diff
@@ -0,0 +1,75 @@
+import { strict as assert } from 'assert';
+import migration60 from './060';
+
+describe('migration #60', function () {
+  it('should update the version metadata', function (done) {
+    const oldStorage = {
+      meta: {
+        version: 59,
+      },
+      data: {},
+    };
+
+    migration60
+      .migrate(oldStorage)
+      .then((newStorage) => {
+        assert.deepEqual(newStorage.meta, {
+          version: 60,
+        });
+        done();
+      })
+      .catch(done);
```

Comment:
Nit: any reason not to use `async/await` here? We have been for the migration tests for a long while now

Reply:
No reason in particular, this has been update to use `async/await`